### ISSUE TITLE
Unnecessary audio file loading

### ISSUE THREAD

**Issue by tylorr** [craftyjs/Crafty#348]
I noticed that if I run:

```
Crafty.audio.add({
  spin: [
    'spin.mp3',
    'spin.ogg',
    'spin.wav'
  ]
});
```

Crafty loops over each file and creates an audio element for each one. If my browser supports all three files formats, it will load each file but only ever play the wav file because it is last in the list.

I feel like Crafty should only load the first file in the list that is compatible with the browser.

**Comment by tylorr** [comment]
Accidental close.

**Comment by starwed** [comment]
I have this fixed in a reworking of the audio code; I need to finish testing it properly, but wanted to give a heads up that I was working on this.

**Comment by starwed** [comment]
Fixed by #487
